admin/partials/cover_art.php

```diff
@@ -0,0 +1,81 @@
+<p class="description">
+	<?php printf(
+		wp_kses(
+			__( 'You can set one or more <a href="%s">cover art</a> images below.', 'apple-news' ),
+			array( 'a' => array( 'href' => array() ) )
+		),
+		esc_url( __( 'https://developer.apple.com/library/content/documentation/General/Conceptual/Apple_News_Format_Ref/CoverArt.html', 'apple-news' ) )
```

Comment:
My theory here was that if the content was available in a translated form, the URL could be translated to point at the URL of the localized content. I'll use a static string for now, though, and we can revisit if we'd like to apply this logic consistently.